integrations/hello-world-ng5/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "hello-world-ng6-ts2",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "npx": "npx",
+    "start": "npm install @ngxs/store && npx -- ng build"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "5.2.0",
+    "@angular/common": "5.2.0",
+    "@angular/compiler": "5.2.0",
+    "@angular/core": "5.2.0",
+    "@angular/forms": "5.2.0",
+    "@angular/http": "5.2.0",
+    "@angular/platform-browser": "5.2.0",
+    "@angular/platform-browser-dynamic": "5.2.0",
+    "@angular/router": "5.2.0",
+    "core-js": "2.4.1",
+    "rxjs": "5.5.6",
+    "zone.js": "0.8.19"
+  },
+  "devDependencies": {
+    "@angular/cli": "1.7.4",
+    "@angular/compiler-cli": "5.2.0",
+    "@angular/language-service": "5.2.0",
+    "@types/jasmine": "2.8.3",
+    "@types/jasminewd2": "2.0.2",
+    "@types/node": "6.0.60",
+    "codelyzer": "4.0.1",
+    "jasmine-core": "2.8.0",
+    "jasmine-spec-reporter": "4.2.1",
+    "karma": "2.0.0",
+    "karma-chrome-launcher": "2.2.0",
+    "karma-coverage-istanbul-reporter": "1.2.1",
+    "karma-jasmine": "1.1.0",
+    "karma-jasmine-html-reporter": "0.2.2",
+    "npx": "^10.2.0",
+    "protractor": "5.1.2",
+    "ts-node": "4.1.0",
+    "tslint": "5.9.1",
+    "typescript": "2.5.3"
```

Comment:
typescript version should be 2.7, see #847

Reply:
We need Angular 5 + TypeScript 2.5